frontend/src/pages/candidate.js

```diff
@@ -143,7 +143,7 @@ class CandidatePage extends Component<Props> {
                 color="primary"
                 onClick={() => this.handleShowAllInterviews(candidate._id)}
               >
-                Show Candidate Interviews
+                Show Interviews
```

Comment:
Was this just a spacing thing?

Reply:
yeah, that and I thought it was pretty verbose and unnecessary since the button is in a specific Candidate's page and it's implicit that the button will show the candidate's interviews